docs/examples/201/key-vault-secret-create/main.bicep

```diff
@@ -116,29 +116,12 @@ resource vault 'Microsoft.KeyVault/vaults@2019-09-01' = {
   }
 }
 
-// workaround for missing loop support - just using the first secret for now
-var firstSecretName = first(secretsObject.secrets).secretName
-var firstSecretValue = first(secretsObject.secrets).secretValue
-
-resource secret 'Microsoft.KeyVault/vaults/secrets@2019-09-01' = {
-  name: '${vault.name}/${firstSecretName}'
+resource secrets 'Microsoft.KeyVault/vaults/secrets@2018-02-14' = [for secret in secretsObject.secrets: {
```

Comment:
Can this be simplified to:
```bicep
resource secrets 'Microsoft.KeyVault/vaults/secrets@2018-02-14' = [for secret in secretsObject.secrets: {
  name: '${vault.name}/${secret.secretName}'
  properties: {
    value: secret.secretValue
  }
}]
```

Reply:
Will take a look in the next one.

Reply:
Took a look - yeah can totally be simplified.